### PR TITLE
og:image and og:url meta tags

### PR DESCRIPTION
**Pull request by sachinmjadhav**
Added image and url meta tags to all pages.

### CODE REVIEW

File: src/pages/careers.mdx [hunk 1]
```diff
@@ -1,5 +1,14 @@
 import Helmet from 'react-helmet'
 import { InputButton } from 'bricks'
+import SEO from './../components/seo'
+
+<SEO
+  title="Careers | Codebrahma"
+  description="We don't offer jobs. We offer CAREERS."
+  keywords={["ruby on rails", "Ruby on Rails development", "angularjs framework", "angularjs code", "meteor js", "react js", "reactjs native", "node js development", "node js website", "spree commerce", "ruby on rails developer", "react js components", "javascript meteor", "Angular javascript", "angular framework", "js angular", "angular website", "Android Website", "ios app development", "mobile web app", "javascript android app", "web application development company"]}
+  image="https://codebrahma.com/static/40e053c59830488fb86e5427205788cf/6a612/react-colored.png"
```
Comment (Kameshwaran) [inline src/pages/careers.mdx]: I dont think this is a right way to do it.

**Review by sanandnarayan — CHANGES_REQUESTED** [review]
Can you get @Kameshwaran approval on this, even I am not sure of the right way to do this

File: src/pages/careers.mdx (hunk 1 above)
Comment (sachinmjadhav) [reply src/pages/careers.mdx]: added dynamic links for og:image

File: src/pages/index.js [hunk 2]
```diff
@@ -1,93 +1,107 @@
...
-    <Flex flexDirection='column'>
-      <Box alignSelf={['center', 'flex-start']} width={1}>
-        <ReactIcon width="200" height="200" />
+const IndexPage = ({ data }) => {
+  
+  const structuredData = `{
+    "@context" : "http://schema.org",
+    "@type" : "LocalBusiness",
+    "name" : "Codebrahma",
+    "image" : "https://codebrahma.com${data.rippling_screenshot.publicURL}",
```
Comment (Kameshwaran) [inline src/pages/index.js:19]: @sanandnarayan do you think it is right to place rippling screenshot as the image for Codebrahma website? should not this be Codebrahma logo?

**Review by Kameshwaran — APPROVED** [review]

**Comment by Kameshwaran** [comment]
PR looks good to me

**Comment by Kameshwaran** [comment]
@sanandnarayan please merge this.

**Review by sanandnarayan — APPROVED** [review]

**Pull request merged**